pxtblocks/blocklyloader.ts

```diff
@@ -2286,6 +2286,7 @@ namespace pxt.blocks {
         // Dropdown menu of variables_get
         msg.RENAME_VARIABLE = lf("Rename variable...");
         msg.DELETE_VARIABLE = lf("Delete the \"%1\" variable");
+        msg.DELETE_VARIABLE_CONFIRMATION = lf("Delete %1 uses of the \"%2\" variable?");
```

Comment:
Thanks @sgk for the change. I know this is how Blockly words the confirmation, could we perhaps use this opportunity to come up with a better way to word this? I feel like the language used here is a little pro dev. @ganicke @pelikhan thoughts?

Reply:
No opinion on this.

Reply:
At this I propose to get it in. We need this fix in the next release iteraiton of the micro:bit.

Reply:
Something like: "The \"%2\" variable is used %1 times, delete it?"